src/test/java/healthy/lifestyle/backend/users/controller/AuthControllerTest.java

```diff
@@ -99,6 +99,7 @@ void signupTest_shouldReturn201Created() throws Exception {
         User user = dataHelper.getUserById(signupResponseDto.getId());
 
         assertThat(user.getCountry().getId()).isEqualTo(country.getId());
+        assertThat(user.getAge()).isEqualTo(signupRequestDto.getAge());
```

Comment:
`AuthController` requires some refactoring, since now we have more cases for testing taking into account a new `age` field. So, let's check cases where an age is out of a range, and ensure that a controller returns an expected error message and status code.